store/rootmulti/store.go

```diff
@@ -972,6 +972,26 @@ func (rs *Store) buildCommitInfo(version int64) *types.CommitInfo {
 	}
 }
 
+// RollbackToVersion delete the versions after `target` and update the latest version.
+func (rs *Store) RollbackToVersion(target int64) int64 {
+	latest := getLatestVersion(rs.db)
+	i := latest
+	for ; i > target; i-- {
+		rs.pruneHeights = append(rs.pruneHeights, i)
+	}
+	rs.pruneStores()
+
+	if i < target {
```

Comment:
Shouldn't this be `<=`?

Reply:
oh, I think it should be `i < latest`, it's for updating the latest height only if when it changed.